### PR TITLE
Document disabling GPUs

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
From the discussion in #1834

### CODE REVIEW

**Comment by DhairyaLGandhi** [comment]
@IanButterworth  your fork doesn't show up in the "list of forks" in the new PR menu. I wanted to PR this to your fork instead.

File: docs/src/gpu.md [hunk 1]
```diff
@@ -85,3 +85,19 @@ julia> x |> cpu
  ⋮
  0.192538
 ```
+
+## Disable CUDA (or choose which GPUs are visible to Flux)
```
Comment (IanButterworth) [inline docs/src/gpu.md:89]:
```suggestion
## Disabling CUDA or choosing which GPUs are visible to Flux
```

**Review by IanButterworth — COMMENTED** [review]
This is great, thanks! Just a wording/format suggestion

**Review by darsnack — APPROVED** [review]

**Comment by IanButterworth** [comment]
Embarrassingly I didn't test this during review, but it doesn't appear to work for me due to a CUDA.jl issue
https://github.com/JuliaGPU/CUDA.jl/issues/1336

**Pull request merged**